src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.icon-search {
+  position: absolute;
+}
+
+.big-icon {
+  padding-left: 26px;
+  padding-top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.small-icon {
+  padding-left: 13px;
+  padding-top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-bar {
+  padding: 0;
+  outline: none;
+  width: 100%;
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: all 0.3s ease-in-out;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.search-bar::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.big-bar {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-bar {
```

Comment:
check this checklist point
<img width="760" alt="image" src="https://user-images.githubusercontent.com/102411130/223048857-f1f49807-a2f9-4c06-818d-4b870c464f00.png">

Reply:
moved outline below

Reply:
i dont understand. What is incorrect?